scala/scala-impl/src/org/jetbrains/plugins/scala/project/Versions.scala

```diff
@@ -103,7 +105,7 @@ object Versions  {
       Version("0.2.0"),
       Seq("0.2.0-RC1"))
 
-    val Hydra = Entity("https://repo.triplequote.com/artifactory/ivy-releases/com.triplequote/",
+    val Hydra = Entity("/ivy-releases/com.triplequote/",
```

Comment:
Drop the leading `/` since you want to append this path to a URL (i.e., we are defining a relative path here). This is particularly relevant for `new URL(<url>, <path>)` to work as expected. See https://docs.oracle.com/javase/7/docs/api/java/net/URL.html#URL(java.net.URL,%20java.lang.String)